cli/azd/pkg/infra/provisioning/bicep/prompt.go

```diff
@@ -75,6 +79,68 @@ func autoGenerate(parameter string, azdMetadata azure.AzdMetadata) (string, erro
 	return genValue, nil
 }
 
+func (a *BicepProvider) locationsWithQuotaFor(
+	ctx context.Context, subId string, locations []string, quotaFor string) ([]string, error) {
+	var sharedResults sync.Map
+	var wg sync.WaitGroup
+	for _, location := range locations {
+		wg.Add(1)
+		go func(location string) {
+			defer wg.Done()
+			results, err := a.azureClient.GetAiUsages(ctx, subId, location)
+			if err != nil {
+				return
+			}
```

Comment:
Should we also add some error handling here?

Reply:
only log. We don't want to stop the operation